src/state/store.js

```diff
@@ -29,23 +29,20 @@ export default (incomingStore = {}, schemas = {}, options = {}) => {
   const state$ = makeSubject();
   const updateIntercepts = [];
 
-  const ajv = new Ajv();
-  const ajvSchemas = new Map(
-    Object.entries(schemas).map(([mountPoint, schema]) => [
-      mountPoint,
-      ajv.compile(schema)
-    ])
+  const schemasMap = new Map(
+    Object.entries(schemas).map(([mountPoint, schema]) => [mountPoint, schema])
```

Comment:
Why do you have to map through an identity function here? Isn't this equivalent to just `new Map(Object.entries(schemas))`?

Reply:
yep good point. just a bad refactoring there.